### PR TITLE
#7 Make the OS index name configurable

### LINKED ISSUE #7: Make the `geonames` open search index name configurable using env variable

### PR DESCRIPTION
**Pull request by chirag4semandex**
Closes #7

### CODE REVIEW

File: mordecai3/elastic_utilities.py [hunk 1]
```diff
@@ -11,17 +11,17 @@
 logger = logging.getLogger(__name__)
 logger.addHandler(logging.NullHandler())
 
-def make_conn(host:str = 'localhost', port:int = 9200):
+def make_conn(host:str = 'localhost', port:int = 9200, index_name:str = 'geonames'):
```
Comment (ianrandman) [inline mordecai3/elastic_utilities.py:14]: Convention is a space before the type hint:
`index_name : str = 'geonames'`

**Review by ianrandman — APPROVED** [review]

File: mordecai3/geoparse.py [hunk 2]
```diff
@@ -169,7 +169,8 @@ def __init__(self,
                  trim=None,
                  check_es=True,
                  os_host='localhost',
-                 os_port=8502):
+                 os_port=8502,
+                 index_name: str = 'geonames'):
```
Comment (djmakwana) [inline mordecai3/geoparse.py:173]: can we keep default name at one place?

**Review by djmakwana — APPROVED** [review]
we should probably rename the `elastic_utillities.py` file also

File: mordecai3/geoparse.py (hunk 2 above)
Comment (chirag4semandex) [reply mordecai3/geoparse.py]: Sure, done

**Pull request merged**